notebooks/openvino_inference.ipynb

```diff
@@ -0,0 +1,225 @@
+{
```

Comment:
<p>"wit" ought to be "with".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/setfit/pull/214/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>thanks</p>